packages/mapviewer/tests/cypress/tests-e2e/featureSelection.cy.js

```diff
@@ -248,6 +250,55 @@ describe('Testing the feature selection', () => {
                 expect(layer.length).to.eq(1)
             })
         })
+        it('Adds pre-selected features and verifys the translation of the feature text after changing the language', () => {
+            cy.log('Open the map with a feature preselected in english')
+            goToMapViewWithFeatureSelection(FeatureInfoPositions.DEFAULT)
+            checkFeatures()
+            checkFeatureInfoPosition(FeatureInfoPositions.BOTTOMPANEL)
+
+            cy.wait(`@htmlPopup`)
+
+            cy.get('[data-cy="feature-list-inner"]').find('strong').contains('Kultur Gueter')
```

Comment:
Here too, maybe not using `<strong>` but have some `data-cy` would make it more resilient to change